src/lib/discord/commands/login.ts

```diff
@@ -0,0 +1,29 @@
+import { createCommand } from './../../discord'
+import type { InteractionReplyOptions } from 'discord.js'
+
+async function handler(interaction): Promise<InteractionReplyOptions | string>  {
+  const { user } = interaction
+
+  if (user.bot) {
+    return 'This command does not support bots logging in.'
+  }
+
+  return {
+    content: `http://localhost:3000/profile/link`,
```

Comment:
```suggestion
    content: `${import.meta.env.VITE_HOST}/profile/link`,
```

can this be used? we can check by building the server (`pnpm build:server`) and looking for this file. It should be statically replaced with the local VITE_HOST env var -- likely `http://localhost:3000`